_includes/footer.html

```diff
@@ -35,12 +35,13 @@
                     </ul>
                 </div>
                 <div class="medium-2 columns end">
-                    <h6>We’re on social media</h6>
+                    <h6>OME Public</h6>
                     <ul class="menu vertical">
                         <li><a href="https://twitter.com/openmicroscopy" target="_blank"><i class="fa fa-fw fa-twitter"></i> Twitter</a></li>
                         <li><a href="https://www.linkedin.com/company/ome---open-microscopy-environment" target="_blank"><i class="fa fa-fw fa-linkedin"></i> LinkedIn</a></li>
                         <li><a href="https://www.facebook.com/openmicroscopy" target="_blank"><i class="fa fa-fw fa-facebook"></i> Facebook</a></li>
                         <li><a href="http://webchat.freenode.net/?channels=%23ome&uio=d4" target="_blank"><i class="fa fa-fw fa-comments"></i> IRC</a></li>
```

Comment:
Outwith this PR, interesting that GitHub highlights a character here. Are we meant to encode using `&amp;`?

Reply:
It is indeed not the only example and AFAIK all the relevant links work fine despite the highlighting

Reply:
https://stackoverflow.com/questions/3705591/do-i-encode-ampersands-in-a-href suggests it *should* be encoded; perhaps our browsers are simply kindly working around the brokenness. I can create a card.

Reply:
Maybe it's something odd with our local networking: whenever I look at the diff on GH then for this image I get,
![social](https://user-images.githubusercontent.com/2630707/30856855-9dcb2b56-a2b1-11e7-98a7-a7aeb33c5d8e.png)

Reply:
Loads fine for me!